src/store/userState.js

```diff
@@ -45,11 +45,24 @@ class UserState {
       localStorage.setItem("AUTH_DATA", JSON.stringify(data));
     } catch (e) {
       runInAction(() => {
-        this.data = e;
+        this.error = e;
         this.loading = false;
       });
     }
   }
+
+  logout = async () => {
+    try {
+      this.loading = true;
+      await logout();
+      localStorage.clear();
+      window.location.reload();
+      this.loading = false;
+    } catch (e) {
+      this.loading = false;
```

Comment:
@ikadushkina you write `this.loading = false` in both try and catch blocks. You can use `finally` block to not duplicate code in the succeeded try and catch. For now it's not so critical as saves only 1 line of duplication. But if the logic shared between try and catch will grow - keep it in mind.